### PR TITLE
[ICDS-Dashboard] Retry getting data if delay in calculation by aggregation script

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @calellowitz @Rohit25negi 

Regarding this ticket: https://dimagi-dev.atlassian.net/browse/ICDS-547 I added functionality to try to get data when we have a delay in aggregation script. Currently, we try 3 times in 5 seconds delay between them.

Need QA: no
Environment: ICDS
locally tested: yes (also I added tests which simulate that for the first time when we try to get data we have 0s)

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

File: custom/icds_reports/migrations/0001_squashed_0110_aww_incentive_report_monthly.py [hunk 1]
```diff
@@ -1,1887 +0,0 @@
-# -*- coding: utf-8 -*-
```
Comment (calellowitz) [inline custom/icds_reports/migrations/0001_squashed_0110_aww_incentive_report_monthly.py]: why is this deleted?

**Review by calellowitz — CHANGES_REQUESTED** [review]
One question, otherwise looks fine

File: custom/icds_reports/migrations/0001_squashed_0110_aww_incentive_report_monthly.py (hunk 1 above)
Comment (lwyszomi) [reply custom/icds_reports/migrations/0001_squashed_0110_aww_incentive_report_monthly.py]: Sorry, @calellowitz I deleted it by mistake. I restored it and PR is ready to merge

**Review by calellowitz — APPROVED** [review]
